lib/graticule/geocoder/mapbox.rb

```diff
@@ -13,76 +14,27 @@ def locate(address)
 
       protected
 
-      # A result looks like:
-      # [
-      #   {
-      #     "id"=>"address.110173544177",
-      #     "lon"=>-122.02912,
-      #     "lat"=>37.33054,
-      #     "name"=>"1 Infinite Loop",
-      #     "type"=>"address"
-      #   }, {
-      #     "id"=>"mapbox-places.95014",
-      #     "name"=>"Cupertino",
-      #     "type"=>"city"
-      #   }, {
-      #     "id"=>"province.1112151813",
-      #     "name"=>"California",
-      #     "type"=>"province"
-      #   }, {
-      #     "id"=>"country.4150104525",
-      #     "name"=>"United States",
-      #     "type"=>"country"
-      #   }
-      # ]
       class Result
         attr_accessor :lat, :lon, :address, :city, :province, :country, :precision, :postal_code
 
         def initialize(input)
```

Comment:
Not really related to the changes in this PR, but is there a better name for this? `input` isn't clear to me.

Reply:
Good point, I'll change that.